Library/Homebrew/test/dev-cmd/livecheck_spec.rb

```diff
@@ -20,7 +20,28 @@
       .and be_a_success
   end
 
-  it "gives an error when no arguments are given and there's no watchlist", :integration_test do
+  it "errors when a `livecheck` block URL symbol does not correspond to a checkable URL", :integration_test do
```

Comment:
Would rather not add a dedicated integration test for this but instead make it a unit test below.

Reply:
Makes sense. With moving the `#raise` into `#livecheck_url_to_string`, are the unit tests for `#livecheck_url_to_string` enough to cover our needs?

Reply:
I think so yes